Index: src/use_case/back/BackInteractor.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package use_case.back;\n\npublic class BackInteractor {\n}\n
===================================================================
diff --git a/src/use_case/back/BackInteractor.java b/src/use_case/back/BackInteractor.java
--- a/src/use_case/back/BackInteractor.java	
+++ b/src/use_case/back/BackInteractor.java	
@@ -1,4 +1,8 @@
 package use_case.back;
 
-public class BackInteractor {
+public class BackInteractor implements BackInputBoundary{
+    @Override
+    public void execute() {
+
+    }
 }
