Index: src/interface_adapter/list_liked_post/ListLikedPostViewModel.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package interface_adapter.list_liked_post;\n\nimport interface_adapter.ViewModel;\n\nimport javax.swing.text.View;\nimport java.beans.PropertyChangeListener;\nimport java.beans.PropertyChangeSupport;\n\npublic class ListLikedPostViewModel extends ViewModel {\n    private ListLikedPostState state = new ListLikedPostState();\n    private final PropertyChangeSupport support = new PropertyChangeSupport(this);\n    public ListLikedPostViewModel(String viewName) {\n        super(\"liked post\");\n    }\n\n    @Override\n    public void firePropertyChanged() {\n        support.firePropertyChange(\"state\", null, this.state);\n    }\n\n    @Override\n    public void addPropertyChangeListener(PropertyChangeListener listener) {\n        support.addPropertyChangeListener(listener);\n    }\n\n    public ListLikedPostState getState(){\n        return state;\n    }\n    public void setState(ListLikedPostState newstate){\n        this.state = newstate;\n    }\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/interface_adapter/list_liked_post/ListLikedPostViewModel.java b/src/interface_adapter/list_liked_post/ListLikedPostViewModel.java
--- a/src/interface_adapter/list_liked_post/ListLikedPostViewModel.java	
+++ b/src/interface_adapter/list_liked_post/ListLikedPostViewModel.java	
@@ -8,6 +8,9 @@
 
 public class ListLikedPostViewModel extends ViewModel {
     private ListLikedPostState state = new ListLikedPostState();
+    public final String TITLE_LABEL = "Liked Post";
+    public final String BACK_LABEL = "Back to post piazza";
+    public final String SHOW_COMMNET_LABEL = "Show comment";
     private final PropertyChangeSupport support = new PropertyChangeSupport(this);
     public ListLikedPostViewModel(String viewName) {
         super("liked post");
Index: src/use_case/list_liked_post/ListLikedPostOutputData.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package use_case.list_liked_post;\n\nimport interface_adapter.list_liked_post.ListLikedPostInputData;\n\nimport java.util.ArrayList;\nimport entities.postEntity;\npublic class ListLikedPostOutputData {\n    private ArrayList<postEntity> listOfPost;\n    public ListLikedPostOutputData(ArrayList<postEntity> listOfPost){\n        this.listOfPost = listOfPost;\n    }\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/use_case/list_liked_post/ListLikedPostOutputData.java b/src/use_case/list_liked_post/ListLikedPostOutputData.java
--- a/src/use_case/list_liked_post/ListLikedPostOutputData.java	
+++ b/src/use_case/list_liked_post/ListLikedPostOutputData.java	
@@ -5,8 +5,11 @@
 import java.util.ArrayList;
 import entities.postEntity;
 public class ListLikedPostOutputData {
-    private ArrayList<postEntity> listOfPost;
+    private final ArrayList<postEntity> listOfPost;
     public ListLikedPostOutputData(ArrayList<postEntity> listOfPost){
         this.listOfPost = listOfPost;
     }
+    public ArrayList<postEntity> getListOfPost(){
+        return listOfPost;
+    }
 }
Index: src/interface_adapter/list_liked_post/ListLikedPostState.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package interface_adapter.list_liked_post;\n\nimport java.util.ArrayList;\nimport entities.postEntity;\npublic class ListLikedPostState {\n    private ArrayList<postEntity> listOfLikedPost = new ArrayList<postEntity>();\n    private String postError = null;\n    public ListLikedPostState(ListLikedPostState listLikedPostState){\n        this.listOfLikedPost = listLikedPostState.listOfLikedPost;\n    }\n    public ListLikedPostState(){}\n    public ArrayList<postEntity> getListOfLikedPostArray(){\n        return listOfLikedPost;\n    }\n\n    public void setPostError(String postError) {\n        this.postError = postError;\n    }\n    public String getPostError(){\n        return this.postError;\n    }\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/interface_adapter/list_liked_post/ListLikedPostState.java b/src/interface_adapter/list_liked_post/ListLikedPostState.java
--- a/src/interface_adapter/list_liked_post/ListLikedPostState.java	
+++ b/src/interface_adapter/list_liked_post/ListLikedPostState.java	
@@ -2,11 +2,14 @@
 
 import java.util.ArrayList;
 import entities.postEntity;
+import entities.User;
 public class ListLikedPostState {
     private ArrayList<postEntity> listOfLikedPost = new ArrayList<postEntity>();
+    private User user;
     private String postError = null;
-    public ListLikedPostState(ListLikedPostState listLikedPostState){
+    public ListLikedPostState(ListLikedPostState listLikedPostState, User user){
         this.listOfLikedPost = listLikedPostState.listOfLikedPost;
+        this.user = user;
     }
     public ListLikedPostState(){}
     public ArrayList<postEntity> getListOfLikedPostArray(){
Index: src/interface_adapter/list_liked_post/ListLikedPostPresenter.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package interface_adapter.list_liked_post;\n\nimport use_case.list_liked_post.ListLikedPostOutputBoundary;\nimport use_case.list_liked_post.ListLikedPostOutputData;\n\npublic class ListLikedPostPresenter implements ListLikedPostOutputBoundary {\n    private final ListLikedPostViewModel likedPostViewModel;\n\n\n    public ListLikedPostPresenter(ListLikedPostViewModel likedPostViewModel) {\n        this.likedPostViewModel = likedPostViewModel;\n    }\n\n    @Override\n    public void prepareSuccessView(ListLikedPostOutputData listLikedPostOutputData) {\n\n    }\n\n    @Override\n    public void prepareFailView() {\n\n    }\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/interface_adapter/list_liked_post/ListLikedPostPresenter.java b/src/interface_adapter/list_liked_post/ListLikedPostPresenter.java
--- a/src/interface_adapter/list_liked_post/ListLikedPostPresenter.java	
+++ b/src/interface_adapter/list_liked_post/ListLikedPostPresenter.java	
@@ -1,19 +1,25 @@
 package interface_adapter.list_liked_post;
 
+import interface_adapter.ViewManagerModel;
 import use_case.list_liked_post.ListLikedPostOutputBoundary;
 import use_case.list_liked_post.ListLikedPostOutputData;
 
 public class ListLikedPostPresenter implements ListLikedPostOutputBoundary {
     private final ListLikedPostViewModel likedPostViewModel;
+    private ViewManagerModel viewManagerModel;
 
 
-    public ListLikedPostPresenter(ListLikedPostViewModel likedPostViewModel) {
+    public ListLikedPostPresenter(ListLikedPostViewModel likedPostViewModel, ViewManagerModel viewManagerModel) {
         this.likedPostViewModel = likedPostViewModel;
+        this.viewManagerModel = viewManagerModel;
     }
 
     @Override
     public void prepareSuccessView(ListLikedPostOutputData listLikedPostOutputData) {
+        ListLikedPostState listLikedPostState = likedPostViewModel.getState();
 
+        this.viewManagerModel.setActiveView(likedPostViewModel.getViewName());
+        this.viewManagerModel.firePropertyChanged();
     }
 
     @Override
